lib/api/document_controller.ex

```diff
@@ -1,6 +1,6 @@
 defmodule Argos.API.DocumentController do
 
-  @elasticsearch_url Application.get_env(:argos, :elasticsearch_url)
+  @elasticsearch_url Application.get_env(:argos, :elasticsearch_url) <> Application.get_env(:argos, :index_path)
```

Comment:
Ich finde die String interpolation schicker als den Operator um Listen zu verketten, das wäre auch einheitlicher mit dem Rest der Anwendung. https://hexdocs.pm/elixir/String.html

Außerdem könnte man dann die Konfiguration _index_path_ einfach ohne führenden "/" definieren und in _index_name_ umbenennen. Den Pfad baut man sich dann anschließend, indem man entsprechend in der Interpolation den "/" zwischen _elasticsearch_url_ und _index_name_ klemmt.

Reply:
ich verstehe den ersten part nicht ganz. <> ist ja genau das was in deinem link angegeben wird, oder meinst du <> "" <> so etwas ?

Reply:
Ich meinte das hier: 

Interpolation
Strings in Elixir also support interpolation. This allows you to place some value in the middle of a string by using the #{} syntax (..)